app/controllers/bright_events_controller.rb

```diff
@@ -11,19 +11,26 @@ def new
     @event = BrightEvent.new
   end
 
+  def user_events
+    @events = current_user.bright_events
+  end
+
   def create
-    @event = BrightEvent.new(event_params)
+    @event = current_user.bright_events.create(event_params)
 
     if @event.save
-      redirect_to bright_events_path
+      redirect_to user_bright_events_path
     else
       flash[:error] = @event.errors.full_messages.to_sentence
       redirect_to new_bright_event_path
     end
   end
 
-  def edit
-    @event = BrightEvent.find(params[:id])
+  def edit() end
```

Comment:
change this line to: 
```suggestion
  def edit; end
```

Reply:
Done!